README.md

```diff
@@ -14,10 +14,11 @@ where you can enjoy your happy MySQL administration.
 
 ## Credentials
 
-phpMyAdmin does use MySQL server credential, please check the corresponding
-server image for information how it is setup.
+phpMyAdmin connects using your MySQL server credentials, please check the corresponding
+database server image for information on what the default username and password are
+or to modify these during installation.
```

Comment:
I had to read it multiple times to understand the `are or` could you split this in two phrases please ?

Reply:
Good suggestion, this is a bit too awkward.